apps/src/music/views/HeaderButtons.tsx

```diff
@@ -73,7 +79,10 @@ const HeaderButtons: React.FunctionComponent<HeaderButtonsProps> = ({
       <button
```

Comment:
Why not actually disable the button via the `disabled` attribute?

Reply:
Good call.  Will use this for the undo and redo buttons.